doc/nrf/libraries/nrf_rpc/nrf_rpc_ipc.rst

```diff
@@ -0,0 +1,23 @@
+.. _nrf_rpc_ipc_readme:
+
+nRF RPC IPC Service transport
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF RPC IPC Service transport allows you to use the :ref:`nrf_rpc` library to execute procedures on remote processors.
+It uses the IPC Service no-copy mechanism. 
+
+The nRF RPC IPC Service transport is used in the :ref:`ble_rpc` library and also in the :ref:`nrf_rpc_entropy_nrf53` sample.
```

Comment:
```suggestion
This feature is used in the :ref:`ble_rpc` library and in the :ref:`nrf_rpc_entropy_nrf53` sample.
```
A bet too much repetition if we mention the whole name again. Can we call it a feature?

Reply:
We can. I changed it

Reply:
Sure. Done